packages/javascript-sdk/src/fr-auth/callbacks/attribute-input-callback.ts

```diff
@@ -51,7 +51,11 @@ class AttributeInputCallback<T extends string | number | boolean> extends FRCall
    * Gets the callback's failed policies.
    */
   public getFailedPolicies(): PolicyRequirement[] {
-    return this.getOutputByName<PolicyRequirement[]>('failedPolicies', []);
+    const failedPolicies = this.getOutputByName<PolicyRequirement[]>(
+      'failedPolicies',
+      [],
+    ) as unknown as string[];
+    return failedPolicies.map((v) => JSON.parse(v)) as PolicyRequirement[];
```

Comment:
Do we want to parse this without a try-catch? If don't use a try-catch, we won't gracefully handle any edge cases.